karabo_data/tests/conftest.py

```diff
@@ -55,6 +55,17 @@ def mock_fxe_run():
         make_examples.make_fxe_run(td)
         yield td
 
+@pytest.fixture(scope='module')
+def mock_spb_run():
+    with TemporaryDirectory() as td:
+        make_examples.make_spb_run(td)
+        yield td
+
+@pytest.fixture(scope='module')
+def mock_spb_run():
```

Comment:
I think these should have different names, otherwise only one of them can be used. Maybe `mock_spb_run_raw` and `mock_spb_run_proc`?

Reply:
I thought one might only check one run (fxe/spb) for proc and the other (spb/fxe) for raw. Having to two fixtures is of course nonsense. I'll delete one

Reply:
Yup, that's also fine to have a calibrated SPB run and a raw FXE run for testing. I'd still suggest naming the new one with `proc` or `calib` in the name to make that clear.

Reply:
I've changed the fixture names for mock_fxe/spb_run and added raw/proc to the method name depending on what they kind of data they represent. But I think that will clash with the other PR I've just seen. You want me to rename mock_fxe_raw_run back to mock_fxe_run? If done then I also would like to rename mock_spb_proc_run back to mock_spb_run for consitency.

Reply:
No, that's OK, the new name makes sense. We'll just have to reconcile the difference after we merge one of the PRs.